poc_mobile_verifier/src/scheduler.rs

```diff
@@ -10,6 +9,12 @@ pub struct Scheduler {
     pub reward_period: Range<DateTime<Utc>>,
 }
 
+// I don't want to have to define this struct, but otherwise I'd have to
+// import the deprecated time crate.
+#[derive(thiserror::Error, Debug)]
```

Comment:
You could import Tokio's time::Duration right? I know it's just a re-export.

Alternately just use std::time::Duration already.. We'll have a lot to update regardless when that deprecation finally hits?

Reply:
We have to go from chrono -> std -> tokio time